src/triage/cli.py

```diff
@@ -287,6 +270,17 @@ def __call__(self, args):
         elif args.validate:
             self.experiment.validate()
             self.experiment.run()
+        elif args.show_timechop:
```

Comment:
It seems the intention is to add to the existing "don't run" option, `validate_only`, a new one, `show_timechop`.

I don't know if changing the `validate` default was intended to help with this; though, I don't think our desire to validate, by default, has changed.

This allows for what I think would be nonsensical invocations, however, _e.g._:

    --validate --show-timechop

...which won't show any timechop, no?

If I'm right, I'd recommend:

1) Don't change the defaults, (if they still hold for how we want to run the experiment).
2) Move this `elif` to check for `show_timechop` _first_ -- it's not a default, so if it's set, they definitely wanted it
3) Add some of these arguments exactly the same, but to a mutually-exclusive group, (rather than directly onto the parser) [[docs]](https://docs.python.org/3.7/library/argparse.html#mutual-exclusion). I would think `('--validate', '--validate-only', '--show-timechop')`, for example, are such a mutually-exclusive group -- it doesn't make sense for the user to mix these on the command line.